applications/nrf_desktop/README.rst

```diff
@@ -1709,6 +1709,9 @@ The B0 bootloader requires the following options enabled:
 * :kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE` - Required for providing the signature used for image signing and verification.
 * :kconfig:option:`CONFIG_FW_INFO` - Required for the application versioning information.
 * :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` - Enable this option to set the version of the application after you enabled :kconfig:option:`CONFIG_FW_INFO`.
+  The nRF Desktop application with the B0 bootloader configuration builds two application images: one for the SO slot and the other for the S1 slot.
```

Comment:
TBC - shouldn't it say S0 slot? I think the letter 'o' is used here instead

Reply:
Thanks, you are right, I will fix it in the next patch:

```suggestion
  The nRF Desktop application with the B0 bootloader configuration builds two application images: one for the S0 slot and the other for the S1 slot.
```